### ISSUE TITLE
Mapping revision not correctly injected in tzdb2020a.nzd

### ISSUE THREAD

**Issue by mnivet** [nodatime/nodatime#1559]
The revision number seem to have been badly injected in this file
This is visible in the VersionId property of an IDateTimeZoneProvider loaded from that file which is "TZDB: 2020a (mapping: $Revision$)" while a number would be expected instead of that $Revision$
This is also visible on that page: [https://nodatime.org/TimeZones?version=2020a#](https://nodatime.org/TimeZones?version=2020a#)

It' not a big deal but it's just visually annoying for me.

**Comment by jskeet** [comment]
Ooh, that's odd. Thanks, will fix. I'll probably leave the 2020a file as it is, but get it fixed for 2020b. No idea what's gone wrong there... it looks like it was okay for 2019c.

**Comment by jskeet** [comment]
It turns out this is a CLDR problem that we're faithfully propagating. CLDR v36 has this in its mapping file:

```xml
<version number="$Revision$"/>
```

Currently seeing if CLDR v37 has the same problem...

**Comment by jskeet** [comment]
Nope, CLDR v37 is the same. Will leave this issue open, but I'm not sure of the best approach.

I've filed https://unicode-org.atlassian.net/browse/CLDR-13931 to see if this can be fixed for v38...

**Comment by TorbenJ** [comment]
Is the `$Revision$` value mandatory when using Tzdb or can I just use `TzdbDateTimeZoneSource.Default.TzdbVersion` (which currently gives me `2021a`), i.e. are there revisions of `2021a` (or any future version) or will the next be different e.g. `2021b`?

**Comment by malcolmr** [comment]
From https://unicode-org.atlassian.net/browse/CLDR-13931, it looked like they were planning on removing the version element entirely? (Although it appears to be present in all the CLDR files I can find, contrary to what that comment might suggest.)
